src/puppetlabs/trapperkeeper/services/status/cpu_monitor.clj

```diff
@@ -0,0 +1,86 @@
+(ns puppetlabs.trapperkeeper.services.status.cpu-monitor
```

Comment:
It would be kind of nice to have some docstrings on the functions in this namespace.